mlcomp/parallelm/extra/sagemaker/monitor/report.py

```diff
@@ -8,10 +8,21 @@ class Report(object):
     _last_metric_values = {}
 
     @staticmethod
-    def job_status(job_name, running_time_sec, status):
+    def job_status(job_name, running_time_sec, billing_time_sec, status=""):
         Report._last_metric_values[job_name] = status
-        tbl = Table().name("SageMaker Job Status").cols(["Job Name", "Running Time (sec)", "Status"])
-        tbl.add_row([job_name, str(timedelta(seconds=running_time_sec)), status])
+        tbl = Table().name("SageMaker Job Status").cols(["Job Name", "Total Running Time",
+                                                         "Time for Billing", "Status"])
```

Comment:
Regarding billing time, is it possible also to obtain the cost? (in $$)

Reply:
Probably yes. Need to be studied and be defined as a task.